### DISCUSSION TITLE
Custom styles based on Planetiler data?

### DISCUSSION THREAD

**Discussion by diehl** [onthegomap/planetiler#938] [Q&A] [answered]
In theory, if I build PMTiles using Planetiler, should I be able to apply any of the default OpenMapTiles style specifications straightaway to the data without issue? I've been struggling with other tools to develop a workflow where I can leverage Maptiler Cloud / Maputnik to develop a custom style and then apply it to PMTiles data.

**Comment by msbarry** [comment, marked as answer]
Yes, the default output planetiler generates is from https://github.com/openmaptiles/planetiler-openmaptiles which maptiler folks have made sure matches https://github.com/openmaptiles/openmaptiles almost exactly. I think the only discrepancy left is that openmaptiles abbreviates some road names but planetiler-openmaptiles does not.

**Reply by diehl** [reply]
Awesome - thank you @msbarry !
